lib/registries/addon/branded/discover/route.ts

```diff
@@ -20,9 +20,7 @@ export default class BrandedRegistriesDiscoverRoute extends Route {
             if (provider.id === 'osf') {
                 this.transitionTo('discover');
             } else {
-                const { href, origin } = window.location;
-                const currentUrl = href.replace(origin, '');
-                this.transitionTo('page-not-found', currentUrl);
+                this.transitionTo('page-not-found', window.location.pathname.replace(/^\//, ''));
```

Comment:
Use `notFoundURL` at app/utils/clean-url.ts to clean up the path.

Reply:
If it looks like it's doing a lot, I suggest `slice(1)` to remove the trailing slash.
```suggestion
                this.transitionTo('page-not-found', window.location.pathname.slice(1));
```